### ISSUE TITLE
Bash shebang wrapped incorrectly

### ISSUE THREAD

**Issue by jdghunt** [stkb/Rewrap#129]
If I code the following bash script
```bash
#!/bin/bash
# A comment about this script which is longer than my maximum line width, which is set to 80 characters.

echo "hello"
```
then put the cursor on the second line (i.e. in the real comment line) and press alt-q to reflow the wrapping, I get the following:
```bash
#!/bin/bash A comment about this script which is longer than my maximum line
# width, which is set to 80 characters.

echo "hello"
```
i.e. the shebang line no longer works.

I would expect Rewrap to do the following:
```bash
#!/bin/bash
# A comment about this script which is longer than my maximum line width, which
# is set to 80 characters.

echo "hello"
```

Please can Rewrap be made to understand that the shebang line is a special comment which shouldn't be wrapped? Thanks. This is a minor annoyance in what's otherwise a great tool.

**Comment by stkb** [comment]
This is now fixed in [pre-release version 17.6](https://github.com/stkb/Rewrap/releases/tag/beta).

**Comment by stkb** [comment]
Now also in stable v1.16.2
